autoprotocol/instruction.py

```diff
@@ -9,6 +9,7 @@
 
 # pragma pylint: disable=too-few-public-methods, redefined-builtin
 from .builders import *  # pylint: disable=unused-wildcard-import
+from .constants import PROVISION_MEASUREMENT_MODES
```

Comment:
pasting conversation from DMs here:

Ramu: 
I was following the previous import statement and if I make change, we might have inconsistency. What do you think?
me:
The import above this (from `.builders` is fine using a wildcard import because the code in `protocol.instruction` depends on all 11 classes defined in `.builders`.
We only need a single object, `PROVISION_MEASUREMENT_MODES` from `.constants`.